frontend/src/components/forms/UserForm.tsx

```diff
@@ -147,6 +149,30 @@ const UserForm: React.FC<UserFormProps> = ({ existingUser, saveInfo }) => {
             </FormItem>
           )}
         />
+        <FormField
+          control={userForm.control}
+          name="remindIn"
+          render={({ field }) => (
+            <FormItem className="w-[40%]">
+              <FormLabel>בדיקה תקופתית</FormLabel>
+              <Select onValueChange={field.onChange} defaultValue={field.value}>
+                <FormControl>
+                  <SelectTrigger dir="rtl">
+                    <SelectValue placeholder={field.value || "תבדוק אותי כל שבוע..."} />
+                  </SelectTrigger>
+                </FormControl>
+                <SelectContent dir="rtl">
+                  <SelectItem value="259200">שלושה ימים</SelectItem>
+                  <SelectItem value="604800">שבוע</SelectItem>
+                  <SelectItem value="1209600">שבועיים</SelectItem>
+                  <SelectItem value="1814400">שלושה שבועות</SelectItem>
+                  <SelectItem value="2592000">חודש</SelectItem>
```

Comment:
Use consts for the values

Reply:
you can also reduce the markup by creating an array of label and options. 

[
{ label:שבוע,
value: ONE_WEEK_IN_SECONDS/MILLISECONDS // Whatever it is
}
]

Reply:
you didn't make the values consts.